mk/config.mk

```diff
@@ -1097,6 +1097,21 @@ ifeq (y-y,$(CFG_CORE_PREALLOC_EL0_TBLS)-$(CFG_WITH_PAGER))
 $(error "CFG_WITH_PAGER can't support CFG_CORE_PREALLOC_EL0_TBLS")
 endif
 
+# CFG_PGT_CACHE_ENTRIES defines the number of entries on the memory
```

Comment:
Please mention that `CFG_PGT_CACHE_ENTRIES` is ignored if `CFG_CORE_PREALLOC_EL0_TBLS=y`.